acceptance_tests/features/pages/collection_exercise.py

```diff
@@ -0,0 +1,34 @@
+from acceptance_tests import browser
+from config import Config
+
+
+def go_to(survey):
+    browser.visit(Config.RESPONSE_OPERATIONS_UI_COLLECTION_EXERCISE.format(survey))
```

Comment:
Based on the comment below it would look something like this f'{RESPONSE_OPERATIONS_UI}/surveys/{survey}'

Reply:
We have an option of using either .format or f'{} it makes no difference as to which format we use